hazelcast-hibernate/hazelcast-hibernate4/src/main/resources/META-INF/services/com.hazelcast.NodeInitializer

```diff
@@ -0,0 +1 @@
+com.hazelcast.instance.ExtendedNodeInitializer
```

Comment:
@cszmajda I think we should remove this file, because `NodeInitializer` is hazelcast-2x class https://github.com/hazelcast/hazelcast/blob/maintenance-2.x/hazelcast/src/main/java/com/hazelcast/impl/base/NodeInitializer.java.  Also, why did you need `ExtendedNodeInitializer` ?